frontend/src/lib/components/LemonModal/LemonModal.scss

```diff
@@ -1,57 +1,100 @@
-.LemonModal {
-    .ant-modal-close-x {
-        position: absolute;
-        top: 0.75rem;
-        right: 0.75em;
-        height: auto;
-        width: auto;
-    }
-    .ant-modal-header,
-    .ant-modal-body,
-    .ant-modal-footer {
-        padding: 1rem;
-    }
-    .ant-modal-header,
-    .ant-modal-footer {
-        display: flex;
-        align-items: center;
-        line-height: 1.5rem;
-        border-color: var(--border);
-    }
+.LemonModal__overlay {
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+
+    transition: background-color 250ms ease-out, backdrop-filter 250ms ease-out;
```

Comment:
This feels _very slightly_ too long for a key interactive element. Why not keep it in sync with button transition times (200ms)? Better yet, our transition times should be standardizes as CSS vars

Reply:
Read a lot of UX discussions on this, particularly Apple who argue that the bigger the element, the slower the transition as that gives it the right "weight" but definitely no opinion on what our values should be.

@clarkus side-discussion but perhaps Transitions should be covered in our Design System somewhere so that engineers can make good decisions here based on some sort of agreed basis principles?

Reply:
Yup we haven't worked on transitions much at all other than @lottiecoxon's animation work. It's something we can address for sure, but generally I think we should use transitions sparingly. While they can be a noticeable detail for some users, the end result is that things display more slowly. Too long of a transition can make your product feel clunky. Shorter transitions and a limited animation area are good goals. @lottiecoxon do you want to spend some time on transitions after you wrap up your current component work?

Reply:
There's a media query you can use to disable animations if users have set reduced motion. Maybe something to consider as we think about how/when to use animations https://www.smashingmagazine.com/2020/09/design-reduced-motion-sensitivities/

Reply:
Good point, @pauldambra. Lottie and I were discussing the same thing yesterday regarding skeleton state animations. Something to consider with a design system are layers of configuration based on user preferences. The most popular options are dark mode and high contrast, but our current palette is high contrast by default. An option for limiting motion and animation would be a great thing to consider. Only mentioning color modes because they're often coupled with the reduced motion options for users.